### DISCUSSION TITLE
Quest about playgrounds

### DISCUSSION THREAD

**Discussion by blauebank** [streetcomplete/StreetComplete#3333] [Ideas]
I would like to suggest adding a quest about playgrounds 
https://wiki.openstreetmap.org/wiki/Tag:leisure=playground

There are many playgrounds listed in the Openstreetmap that lack additional information.

Here are some questions that I came up with:

`What is the official name of the playground?`
	Don't know
	Text area to enter the name

`What type of installations are there?`
image list with the ten or twenty most common installations 
see https://wiki.openstreetmap.org/wiki/Key:playground 
	e.g. slide, playhouse, sandpit, basketswing
	selection of several installation via checkbox possible
	

`What type of ground is there? `
	image list with grass, gravel
	surface=*
	only one image can be selected
	

`Is the playground open to everybody?`
	all
	private
	only customers 

`Is the playground fenced? (important for parents with younger kids)`
	yes
	no
	
`What are the opening times? `
	unlimited
	Text area to enter the time 8a.m to 6pm (or time selector)
	
`Is there a minimum age posted?`
	unlimited
	Text area to enter a number
	

`Is there a maximum age posted?`
	unlimited
	Text area to enter a number

`Is the playground accessible by wheelchair?`
	yes
	no
	
	
What do you think about my idea? Does this make sense?

**Comment by matkoniecz** [comment]
> Is the playground open to everybody?

This is implemented already.

> What is the official name of the playground?

Note that vast majority has no name at all. 

I would recommend reading through https://github.com/streetcomplete/StreetComplete/wiki/Adding-new-Quests-to-StreetComplete and if some are not implemented already and passing this requirements - open issue for them by using https://github.com/streetcomplete/StreetComplete/issues/new?assignees=&labels=&template=quest-suggestion.md

**Reply by blauebank** [reply]
The document is relevant if you know already what questions you want to be answered about playgrounds. 

I was trying to get feedback if people think that questions about playgrounds are a good idea and if so, which questions are relevant and which are not (e.g. the color of the slide is not that important).

So my question was not so much about the "how" but about the "if".
Would users want to answer questions about playgrounds? If so, what questions would be relevant?

**Reply by andrewharvey** [reply]
> Would users want to answer questions about playgrounds? If so, what questions would be relevant?

- Available equipment (installations) is highly desirable.
- Name, useful. Many will have no name, but I think that's okay to tag as noname=yes.
- Type of ground I feel would be hard to answer because it might vary a lot among the equipment within the playground, and probably less useful.
- Access is useful, and already asked.
- Fenced, yes quite desirable.
- Opening hours, yes useful, even though most will be 24/7 better to tag it to catch those which aren't.
- Min/max age I've never seen, maybe make it opt in by country? Useful where it exists.
- Wheelchair, would that mean if spectators can access via wheelchair or the equipment is designed for access via wheelchair?

**Comment by HolgerJeromin** [comment]
ref #589 and #3174

**Comment by tordans** [comment]
What are your thoughts on adding a `check_date=<Date>`-Quest on playground equipment? I find those change every other year and adding an easy way to validate their existence would be great.

>"Is this swing still there?"
"Is this springy still there?"
…

Example where I did this manually: https://www.openstreetmap.org/changeset/113511724#map=19/52.47014/13.43586

The main issue I see is, that the playground tagging values list is not very long so only the most common values could easily be translated into such an existence-question. For all other values the quest would need to be disabled until the value is documented better. (Or do you see another solution?)
